### ISSUE TITLE
[Feature Request] Give SteamPlay games there own Category

### ISSUE THREAD

**Issue by Zero86Sk** [ValveSoftware/steam-for-linux#5679]
Instead of SteamPlay games showing in LIBRARY under STEAMOS + LINUX (xxx)
why not give them there own category like (STEAMPLAY) this way the Linux Native games could be seperated from Windows games on Linux.

They would still show up in INSTALLED etc.

so your library could look like this f.e.
STEAMOS + LINUX (356)
STEAMPLAY (1020)
SOFTWARE (13)
VR (6)
MUSIC
VIDEOS
etc

**Comment by P4NCH1** [comment]
Totally agree, And I'd like to add that the "Runs on this computer via Steam Play" sign that is shown in desktop client mode isn't anywhere to see in Big Picture mode, so in dristros like SteamOS you simply can't tell what's a native Linux game and what will run over Steam Play / Proton

**Comment by akien-mga** [comment]
See also ValveSoftware/Proton#744 for additional context/suggestions.

**Comment by hustler77** [comment]
I just wanted to file the same feature request. I have 800+ titles in my lib and a own category proton powered would be so nice.

**Comment by Ae3NerdGod** [comment]
their* had to. sorry

**Comment by byte1024** [comment]
I thought the whole idea is that Proton whitelisted games will be official so they would need two categories and separate them from other official games which would look kind of silly when you think about it.

If you look at it VR has its own category then why not Proton? Well why not Windows, Mac, etc while we're at it =)

**Comment by Ae3NerdGod** [comment]
The idea of proton is to run windows games on Linux. It doesn't seem like it's been that hard to add a filter feature for the game library that honestly has been needed for a while. A filter with check boxes like on Amazon doesn't seem that hard to implement. 
OS:
      Linux
 ✓     Steamplay 
   ✓   Windows (proton)
      Osx

**Comment by DanMan** [comment]
The least I'd expect is to only have the whitelisted games show up under the Linux filter; certainly not just all of them. It makes the Linux library filter completely useless, because it's now factually the same as the general "Games" filter.

**Comment by Zero86Sk** [comment]
@DanMan Not even whitelisted games, the Linux / SteamOS and Mac categories should be reserved for native games on the respective platforms imho. Give SteamPlay its own category and depending on if you chose whitlisted games or all games, have them show up there.

**Comment by lucifertdark** [comment]
I think the ultimate idea is there should be no need of a different category for games that run in Linux natively & for those that run in Linux via Proton, why would you need to know which OS it actually runs in when all it takes to install & play is to hit a single button? 

Right now it's still early days & there's some meddling with code for those brave enough to do it & performance difference in some cases, but within a year or so the OS category will be a thing of the past with performance being equal on all games & OS.

**Comment by daniel-fanjul-alcuten** [comment]
> the OS category will be a thing of the past with performance being equal on all games & OS
That's never going to happen.

**Comment by lucifertdark** [comment]
> > the OS category will be a thing of the past with performance being equal on all games & OS
> > That's never going to happen.

Oh ye of little faith, remember the days when people said Steam would never be on Linux at all?

**Comment by DanMan** [comment]
> I think the ultimate idea is there should be no need of a different category for games that run in Linux natively & for those that run in Linux via Proton, why would you need to know which OS it actually runs in when all it takes to install & play is to hit a single button?

Yeah, that's what I was getting at. If for all intents and purposes a game behaves exactly like an officially released Linux game, does it really matter how it works under the hood in terms of categorization in Steam? I don't think so, but you're free to disagree Zero86Sk, of course. Just adding my 2 cents.

I mean, there are plenty of games released on Linux, which have some sort of DirectX conversion layer already embedded (basically all the Feral ports, for example). So the distinction of "native" games is already a blurry one. What really matters is official support, and since Valve said they're the ones stepping in for those games running under Proton, that's all I need.

**Comment by Zero86Sk** [comment]
@DanMan I can totally see all the points you made. But valve themselves say this is a steamplay feature i wonder if they plan to mark whitelisted games as linux compatable in the store in the future once proton has matured more.

Buit imho having a category that seperates the native and steamplay games would declitter the interface, You would still have Games and Installed etc that would show everything, it would just be an option to just view native games. I see two solutions to this. Either do seperate categories for Steamos and Steamplay or why not drop the category alltogether since you than effectivly have just GAMES with everything.

**Comment by DJViking** [comment]
SteamOS + Linux
SteamPlay + Linux
Mac
Windows
Games (All games)

**Comment by Ruedii** [comment]
Having "Steamplay Whitelisted" games separate from "SteamPlay Unsupported Mode" would be good as well.

Also, having the option in the store to "show SteamPlay whitelisted titles" for users only listing Linux titles would be nice as well.

**Comment by pbhj** [comment]
> why would you need to know which OS it actually runs in when all it takes to install & play is to hit a single button?

Yeah, when we get there then we can argue why that's a bad idea (!) but for now none of the 4 titles I've tried that "work via Proton" (or whatever the wording is) have actually worked at all; so it's still very useful to know when searching for titles to buy/play whether they're running natively or not.

**Comment by Zorrototo** [comment]
I would like this feature too. Having all my Linux + Windows games in the 'SteamOS + Linux' category is just the same as lokking into the 'Games' category, they all are here:
![steamlist](https://user-images.githubusercontent.com/17416415/61724450-4bba0200-ad6e-11e9-8099-9aa14699d76b.jpg)

A 'Windows' or 'Proton' category would be OK.

Alternatively, there is a Python script that can sort games by Proton rating https://github.com/CorruptComputer/ProtonDB-to-Steam-Library it doesn't solve the problem at all but maybe someone smarter than me can modify this script to create a Windows or Proton category only (instead of the various Proton Rating XXX categories) and add the games automatically in it.

**Comment by Protonator** [comment]
I second the request and would like to have separate categories for native Linux games and Proton titles.

**Comment by BigCojones** [comment]
AT LEAST you should be able to gain back the native steam play library and filter function after disabling "Enable steam for all other titles", which it doesn't do. (This would be the easiest implementation)

It only seemingly needs to have proton installed on your system, to lose all function of the steam play filter. As i turned off the enable for other titles and reinstalled steam and still have all games reported as native.
